python_files/ml_on_iris_dataset.py

```diff
@@ -0,0 +1,194 @@
+# Iris Machine Learning Assignment 1
```

Comment:
shebang missing to make this actually executable.

Reply:
@dafrenchyman - Isn't this the shebang "#!/usr/bin/env bash". That is at the top of my script. Did I need to put something else?

Reply:
Yeah,.. the shebang goes at the top of the python files if you want to make them executabe.

Reply:
@dafrenchyman ok so just to clarify I need  "#!/usr/bin/env bash"  in my script and in the files my script calls? Do I need to add chmod +x in front of the python file in the script after I add the shebang to the python script?

Reply:
No... let me explain a little better:
- For bash files you need `#!/usr/bin/env bash`
- For python files you'll need `#!/usr/bin/env python3`

Reply:
The `chmod +x` then converts the files to an executable, so you can then run them directly, but if we didn't specify what they need to be run with (the shebang does that) your system wouldn't know how to run them.

Reply:
ok that makes sense. I know that it worked on my work computer so maybe it worked on accident? haha